titan-server/migrations/2019-02-03-013951_create_excuses_table/up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE user_event_excuses (
```

Comment:
The fields that aren't nullable should be marked as such in the migration. This prevents Diesel from generating schemas that are incompatible with the type system. This is my view of nullable versus not nullable types in this database table:

|Column|Type|Nullable?|
|--|--|--|
|static12_id|INT|No|
|event_type|VARCHAR(255)|No|
|event_date|DATETIME|No|
|comments|TEXT|No|
|ack_user_id|INT|Yes|
|ack_date|DATETIME|Yes|
|ack_comments|TEXT|Yes|
|date_created|DATETIME|No|
|date_modified|DATETIME|No|